File: hackerrank/solve-me-first/solve-me-first.cpp
#include <iostream>

int
solve_me_first(int a, int b) {
    return a + b;
}

int
main() {
    int num1, num2;
    std::cin >> num1 >> num2;
    int sum = solve_me_first(num1, num2);
    std::cout << sum << std::endl;
    return 0;
}

File: hackerrank/01_solve-me-first/solve-me-first.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

def solve_me_first(a, b):
    return a + b

if __name__ == "__main__":
    [num1, num2] = [int(_) for _ in input().strip().split()]
    print(solve_me_first(num1, num2))

# EOF